mimic/rest/maas_api.py

```diff
@@ -477,23 +477,21 @@ def get_check(self, request, tenant_id, entity_id, check_id):
                methods=['PUT'])
     def update_check(self, request, tenant_id, entity_id, check_id):
         """
-        Update an existing check
+        Updates a check in place.
         """
-        checks = self._entity_cache_for_tenant(tenant_id).checks_list
-        newcheck = json.loads(request.content.read())
-        newcheck['entity_id'] = entity_id
-        for k in newcheck.keys():
```

Comment:
The re-encoding everything in utf-8 seems to have been eliminated in this change.  There wasn't a test for it before, but I'm not sure if this would break anything for anyone.  If you happen to know the exact use case and that it wouldn't break anyone using this plugin, that's cool, but otherwise maybe we should put it back?

If so, I will file an issue to add a test case for this.